torch/nn/modules/dropout.py

```diff
@@ -181,6 +181,46 @@ def forward(self, input):
 
 
 class FeatureAlphaDropout(_DropoutNd):
+    r"""Randomly masks out entire channels (a channel is a feature map, 
+    e.g. the :math:`j`-th channel of the :math:`i`-th sample in the batch input 
+    is a tensor :math:`\text{input}[i, j]`) of the input tensor). Instead of 
+    setting activations to zero, as in regular Dropout, the activations are set 
+    to the negative saturation value of the SELU activation function.
+
+    Each element will be masked independently for each sample on every forward 
+    call with probability :attr:`p` using samples from a Bernoulli distribution.
+    The elements to be masked are randomized on every forward call, and scaled
+    and shifted to maintain zero mean and unit variance.
+
+    Usually the input comes from :class:`nn.Conv3d` modules.
+
+    As described in the paper
+    `Efficient Object Localization Using Convolutional Networks`_ ,
+    if adjacent pixels within feature maps are strongly correlated
+    (as is normally the case in early convolution layers) then i.i.d. dropout
+    will not regularize the activations and will otherwise just result
+    in an effective learning rate decrease.
+
+    In this case, :func:`nn.Dropout3d` will help promote independence between
+    feature maps and should be used instead.
+
+    Args:
+        p (float, optional): probability of an element to be zeroed.
+        inplace (bool, optional): If set to ``True``, will do this operation
+            in-place
+
+    Shape:
+        - Input: :math:`(N, C, D, H, W)`
+        - Output: :math:`(N, C, D, H, W)` (same shape as input)
+
+    Examples::
+
+        >>> m = nn.Dropout3d(p=0.2)
+        >>> input = torch.randn(20, 16, 4, 32, 32)
+        >>> output = m(input)
+
+    .. _Self-Normalizing Neural Networks: https://arxiv.org/abs/1706.02515
```

Comment:
you are not using this reference in the text above...

Reply:
Addressed in new update.